petab/v1/distributions.py

```diff
@@ -0,0 +1,233 @@
+"""Probability distributions used by PEtab."""
+from __future__ import annotations
+
+import abc
+
+import numpy as np
+from scipy.stats import laplace, norm, uniform
+
+__all__ = [
+    "Distribution",
+    "Normal",
+    "Uniform",
+    "Laplace",
+]
+
+
+class Distribution(abc.ABC):
+    """A univariate probability distribution.
+
+    This class provides a common interface for sampling from and evaluating
+    the probability density function of a univariate probability distribution.
+
+    The distribution can be transformed by applying a logarithm to the samples
+    and the PDF. This is useful, e.g., for log-normal distributions.
+
+    :param log: If ``True``, the distribution is transformed to its
+        corresponding log distribution (e.g., Normal -> LogNormal).
+        If a float, the distribution is transformed to its corresponding
+        log distribution with the given base (e.g., Normal -> Log10Normal).
+        If ``False``, no transformation is applied.
+    """
+
+    def __init__(self, log: bool | float = False):
+        if log is True:
+            log = np.exp(1)
+        self._log = log
```

Comment:
I find this more readable in `_undo_log` and `__apply_log`
```suggestion
        self._log_base = log
```